include/jemalloc/internal/arena_structs_b.h

```diff
@@ -122,13 +122,22 @@ struct arena_bin_s {
 	extent_t		*slabcur;
 
 	/*
-	 * Heap of non-full slabs.  This heap is used to assure that new
+	 * Heaps of non-full slabs.  These heaps are used to assure that new
 	 * allocations come from the non-full slab that is oldest/lowest in
-	 * memory.
+	 * memory.  The _sized heap contains extents from the sized alloc
+	 * region, whereas the _unsized one contains all others.  We prefer the
+	 * sized-region ones over the unsized ones where we can.
+	 *
+	 * Note that we could get by by simply preferring the sized heaps to the
```

Comment:
Not quite, you'd still want to allocate a new sized extent before using an unsized, yes?

Reply:
Got rid of this whole comment and put a similar message in the extent code, where it makes more sense.